drop-boxes/register-attachments-dropbox/README.md

```diff
@@ -0,0 +1,33 @@
+# Attachments Dropbox
+
+## Expected data structure
+The data structure needs to be a root folder, containing a file `metadata.txt`.
+
+Incoming structure overview:
+
+```
+|-<anything> (top level folder name, normally a time stamp of upload time)
+    |
+    |- metadata.txt
+```
+
+openBIS structure overview:
+
+Attachments are attached to the Q_PROJECT_DETAILS experiment type and its sample type Q_ATTACHMENT_SAMPLE.
```

Comment:
maybe add the sample code schema here in regex? (in the code its Q\w{4}000, this is python specific I guess?)

Reply:
regex added